sympy/matrices/dense.py

```diff
@@ -904,8 +907,11 @@ def col_del(self, i):
         col
         row_del
         """
-        for j in range(self.rows - 1, -1, -1):
-            del self._mat[i + j*self.cols]
```

Comment:
As i saw line of code in row_del `self._mat = ..... ` was done in the same commit( b56bffdf1 ) as code in col_del of `del ...` But surrounding them the entire code of functions was written in different commit ( 6c2244b42 ). So i guess it was intentional, is not it ?